### DISCUSSION TITLE
Can you disable the progress bar for a file download or stop the progress bar once the download has started?

### DISCUSSION THREAD

**Discussion by cletisgipson** [unpoly/unpoly#464] [Q&A] [answered]
In my app the user has the ability to download a file.  I would like the unpoly progress bar to either not happen when the download is initiated or for the progress bar to stop when the download has successfully begun in the browser?

Is this possible with UnPoly

**Comment by triskweline** [comment, marked as answer]
Download links [should not be handled by Unpoly](https://makandracards.com/makandra/504778-unpoly-2-don-t-try-to-download-files-through-ajax-requests).

**Reply by cletisgipson** [reply]
Thanks for the link, it provided exactly what I was looking for.  It also provided a few more tidbits that I found extremely helpful.
